### ISSUE TITLE
Can't locate object method "isa" via package "Parser::MGC::Failure"

### ISSUE THREAD

**Issue by eserte** [LibreCat/Catmandu#389]
t/Catmandu-Fix-Parser.t fails on some of my smoker systems like this:
```
#   Failed test 'syntax errors throw FixParseError'
#   at t/Catmandu-Fix-Parser.t line 177.
# expecting: Catmandu::FixParseError
# found: Can't locate object method "isa" via package "Parser::MGC::Failure" at /usr/home/cpansand/.cpan/build/2022022817/Catmandu-1.2016-1/blib/lib/Catmandu/Util.pm line 517.

#   Failed test 'threw Catmandu::FixParseError'
#   at t/Catmandu-Fix-Parser.t line 200.
# expecting: Catmandu::FixParseError
# found: Can't locate object method "isa" via package "Parser::MGC::Failure" at /usr/home/cpansand/.cpan/build/2022022817/Catmandu-1.2016-1/blib/lib/Catmandu/Util.pm line 517.

#   Failed test 'threw Catmandu::FixParseError'
#   at t/Catmandu-Fix-Parser.t line 208.
# expecting: Catmandu::FixParseError
# found: Can't locate object method "isa" via package "Parser::MGC::Failure" at /usr/home/cpansand/.cpan/build/2022022817/Catmandu-1.2016-1/blib/lib/Catmandu/Util.pm line 517.
# Looks like you failed 3 tests of 65.
t/Catmandu-Fix-Parser.t .................. 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/65 subtests 
```

It's not clear what is causing this problem. I see these failures for various perl versions (5.20.3 .. 5.32.1) and various operating systems (linux, freebsd). It can happen with both older and the newest Parser::MGC release.

**Comment by nics** [comment]
related to this voodoo? https://metacpan.org/release/PEVANS/Parser-MGC-0.21/source/lib/Parser/MGC.pm#L509

**Comment by vpeil** [comment]
@nics probably, yes. I found this article: https://www.effectiveperlprogramming.com/2020/01/use-the-infix-class-instance-operator/

Do you have an idea, how to fix this?

**Comment by nics** [comment]
it's some weird isa issue, i released 1.2019 which will hopefully fix this

**Comment by nics** [comment]
fixed in 1.2019
